src/Components/Browser.JS/src/Platform/Circuits/AutoReconnectCircuitHandler.ts

```diff
@@ -0,0 +1,33 @@
+import { CircuitHandler } from './CircuitHandler';
+export class AutoReconnectCircuitHandler implements CircuitHandler {
+  modal: HTMLDivElement;
+  message: Text;
+
+  constructor(private maxRetries: number = 5, private retryInterval: number = 3000) {
+    this.modal = document.createElement('div');
+    this.modal.className = 'modal';
```

Comment:
Could we give this a more unique class name? People are likely to use `modal` for unrelated things in their own apps. Suggestion: `components-reconnection-modal` or something like that.

Reply:
Alternative idea, could we just have a wellknown id into which we will be injecting content and just provide default markup when such id is not found?

That way I can for example write my markup
```html
<div id="blazor-reconnect-modal" class="my bootstrap styles">
</div>
```
And my styles:
```css
#blazor-reconnect-modal:empty{
  diplay:none;
}
```

And have the content just be injected there by the framework.

Reply:
In fact, I would go even further and if you can't find an element with the appropriate id, I would fallback to console.log or similar, as this forces the visuals into people's app.

Reply:
The "well-known ID" idea sounds good. Maybe we could take it a step further and say:

 * If an element with the well-known ID exists in your document, then we just toggle its visibility based on connected/disconnected status
 * If no such element exists, then at the moment when we first want to display it, we dynamically insert one that contains our default markup, including the hardcoded `style` attribute values.

That way it's super easy to customize, while also keeping it as "not a concept that newcomers have to think about" until they actually do want to customize it.

Reply:
Again, this is not to say @pranavkm should necessarily implement this in time for Preview 3 if there are more pressing matters. I'd be happy enough with just removing the `.css` file from the existing implementation if that's important to meet deadlines.

Though TBH it probably wouldn't be very time-consuming to implement.

Reply:
Oh wait, sorry @javiercn, I just re-read your suggestion and it looks like you already were suggesting exactly what I just said. Apologies for stealing your idea!